### PR TITLE
MT-650: hightlighted content

### PR DESCRIPTION
**Pull request by lha047**
Add component and styling for highlighted content

MT-650

### CODE REVIEW

File: src/svelte/HighlightedContentLink.svelte [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  {/if}
+  <div class="layout-flex layout-flex-col highlighted-text" style="gap:0">
+    {#if heading}
+      <h3 class={`heading ${headingClass}`}>
+        {heading}
+      </h3>
+    {/if}
+    <slot />
+  </div>
+</a>
```
Comment (itemodahl) [inline src/svelte/HighlightedContentLink.svelte:30]: I don't think `div` is a valid content in an anchor element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a . Should we use `button` instead?

File: src/ts/utils.ts [hunk 2]
```diff
@@ -10,3 +10,7 @@ export function toKebabCase(text: string): string {
     .replace(/ /g, '-')
     .toLowerCase();
 }
+export function displayDataTime(lang: string, isoDate?: string): string {
+  if (!isoDate) return '';
+  return new Intl.DateTimeFormat(lang).format(new Date(isoDate));
+}
```
Comment (itemodahl) [inline src/ts/utils.ts:16]: Should the check if isoDate exists happen outside this function. And instead make isoDate mandatory argument?

**Review by itemodahl — CHANGES_REQUESTED** [review]
Good job! A couple of feedbacks you can take a look at.

File: src/svelte/HighlightedContentLink.svelte (hunk 1 above)
Comment (lha047) [reply src/svelte/HighlightedContentLink.svelte:30]: Looks to me like you should not use interactive elements either inside link.

**Pull request merged**